WordPress/src/main/res/drawable/ic_notifications_selected.xml

```diff
@@ -1,14 +1,11 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="32dp"
-    android:height="32dp"
-    android:viewportWidth="32"
-    android:viewportHeight="32">
-  <path
-      android:pathData="M5.357,14.208L6.178,13.387C6.551,13.478 6.891,13.539 7.2,13.576C9.094,13.804 9.814,13.146 9.899,13.061L14.88,8.081C16.645,6.316 21.254,4.967 24.649,8.362C27.921,11.634 26.83,16.03 25.065,18.02L20.133,22.952C19.38,23.705 19.36,24.965 19.46,25.837C19.524,26.397 19.638,26.797 19.638,26.797L18.792,27.643L5.357,14.208ZM17.742,24.118C17.866,23.325 18.177,22.433 18.895,21.715L23.787,16.823C25.091,15.314 25.841,12.028 23.412,9.599C22.116,8.303 20.634,7.931 19.283,8.032C17.862,8.137 16.675,8.761 16.117,9.318L11.136,14.299C10.798,14.637 10.1,15.122 8.926,15.302L17.742,24.118Z"
-      android:fillColor="#999999"
-      android:fillType="evenOdd"/>
-  <path
-      android:pathData="M9.767,20.057C9.694,20.115 9.624,20.178 9.556,20.246C8.563,21.239 8.563,22.85 9.556,23.843C10.55,24.837 12.161,24.837 13.154,23.843C13.222,23.776 13.284,23.706 13.343,23.633L9.767,20.057Z"
-      android:fillColor="#999999"
-      android:fillType="evenOdd"/>
+    android:width="24dp"
```

Comment:
There is no separate "non-selected" version or a version with a badge, is there?

Reply:
There is not. We use [setImageViewSelected](https://github.com/wordpress-mobile/WordPress-Android/blob/884031226c132d214c43a8a1ed35f427a2c1de63/WordPress/src/main/java/org/wordpress/android/ui/main/WPMainNavigationView.kt#L149) to change the color filter for the selected state, and [showBadge](https://github.com/wordpress-mobile/WordPress-Android/blob/884031226c132d214c43a8a1ed35f427a2c1de63/WordPress/src/main/java/org/wordpress/android/ui/main/WPMainNavigationView.kt#L402) for the badging.